scripts/UpdateBuildTools.ps1

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env powershell
```

Comment:
We expect this script and the one below to mostly only be run by the CI, correct?

Reply:
Yes